### PR TITLE
Ported upgrade command from migration plugin

### PR DESCRIPTION
**Pull request by Blackbaud-MatthewMiles**
Added upgrade as an available CLI command.

Moved the upgrade command from https://github.com/blackbaud/skyux-sdk-builder-plugin-migrate and added to list of available CLI commands. I removed the verifyLatestVersion check within upgrade.js since that seemed to only be intended for the plugin. All references should be updated (different folder structure).

This is for Issue #27

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Thanks so much for the PR @Blackbaud-MatthewMiles.  Sorry for the delay.  We've been working through a medium sized rewrite on how our ecosystem does certificate management.

We've chosen to get that released and pause this PR for another week or so.  Once we get the cert work released and let the dust settled, I'll circle back around to review this and get it released.

Thanks again for the patience and PR!

**Comment by Blackbaud-MatthewMiles** [comment]
Sounds good @Blackbaud-BobbyEarl.  Thanks for the update.

**Comment by Blackbaud-MatthewMiles** [comment]
Are y'all still working through cert issues and updates?  Just going to gently bump this one.

**Comment by Blackbaud-BobbyEarl** [comment]
Sorry for the delay @Blackbaud-MatthewMiles... had a few other issues creep in that we needed to address first.  https://github.com/blackbaud/skyux-sdk-cli/pull/52 should be the last PR before we're able to pull in your's.

Thanks for your patience.

**Comment by Blackbaud-BobbyEarl** [comment]
I know our timing was off on this PR @Blackbaud-MatthewMiles.  Are you still in a position to work on it?

**Comment by Blackbaud-MatthewMiles** [comment]
Hey @Blackbaud-BobbyEarl sorry I missed your original review comments.  I should have time to go through them this sprint.

**Comment by Blackbaud-MatthewMiles** [comment]
Failed to update the corresponding install spec test after refactoring it to use cleanup.js.  Standby, I will need to update.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.2.0 (2019-10-01)
```
[the remark this answers is not in the document]
Comment (Blackbaud-MatthewMiles) [reply CHANGELOG.md]: This should be removed.

File: lib/cleanup.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const fs = require('fs-extra');
```
[the remark this answers is not in the document]
Comment (Blackbaud-MatthewMiles) [reply lib/cleanup.js]: I took your guidance and have install.js referencing what is occurring in cleanup.js.  This removes it's dependency on `fs`.  Also due to the new dependency, I changed install.js to also use async/await.  Tests were updated to reflect.

File: lib/sort-utils.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+
```
[the remark this answers is not in the document]
Comment (Blackbaud-MatthewMiles) [reply lib/sort-utils.js]: Moved

File: lib/json-utils.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs-extra');
```
[the remark this answers is not in the document]
Comment (Blackbaud-MatthewMiles) [reply lib/json-utils.js]: Moved

**Comment by Blackbaud-MatthewMiles** [comment]
Okay, it should be updated based on latest review comments with tests updated as well :)

**Comment by Blackbaud-MatthewMiles** [comment]
@Blackbaud-BobbyEarl let me know if any of my adjustments need another pass.  Thanks!

**Pull request merged**